weblate/trans/models/unit.py

```diff
@@ -118,16 +118,42 @@ def same(self, unit, exclude=True):
 
     def order_by_request(self, form_data):
         sort_list_request = form_data.get("sort_by", "").split(",")
-        available_sort_choices = ["priority", "position", "context", "num_words"]
-        sort_list = [
-            choice
-            for choice in sort_list_request
-            if choice.replace("-", "") in available_sort_choices
+        available_sort_choices = [
+            "priority",
+            "position",
+            "context",
+            "num_words",
+            "labels",
```

Comment:
This should probably translate to something like `labels__name`, but it needs to be tested.

Reply:
Correct. I will test this out.